test/Tester/StreamingTests/EHImplicitSubscriptionStreamRecoveryTests.cs

```diff
@@ -117,7 +75,56 @@ public EHImplicitSubscriptionStreamRecoveryTests()
             // send end events
             for (int j = 0; j < streamCount; j++)
             {
-                await producers[j].OnNextAsync(new GeneratedEvent { EventType = GeneratedEvent.GeneratedEventType.End });
+                await producers[j].OnNextAsync(new GeneratedEvent { EventType = GeneratedEvent.GeneratedEventType.Report });
+            }
+        }
+
+        private class Fixture : BaseClusterFixture
```

Comment:
Will move to top of class.  In general, I've started moving private code (including classes) to the end of the file.  I've no issue with making exceptions to this in test code.